common/src/main/java/net/arna/jcraft/api/registry/JCreativeMenuTabRegistry.java

```diff
@@ -207,6 +207,7 @@ static CreativeModeTab createJcraftItemGroup() {
                     entries.accept(JItemRegistry.KQ_COIN.get());
                     entries.accept(JItemRegistry.FOOLISH_SAND_BLOCK.get());
                     // misc
+                    entries.accept(JItemRegistry.TRAINING_DUMMY.get());
```

Comment:
Put this after the hot sand. I put foolish sand and hot sand together on purpose